### DISCUSSION TITLE
Several photometric data in different filters

### DISCUSSION THREAD

**Discussion by JLE64315** [phoebe-project/phoebe2#789] [Questions]
Thank you for your answers, I have a photometric data in three filters, anybody know any code written en Phoebe 2.4 to apply the inverse problem, with the three light curves and a radial velocity curve.

Thank you for your time

**Comment by kecnry** [comment]
There is not currently an example script for this exact scenario, but you should be able to find what you need in:
* [Inverse Problem Tutorial](http://phoebe-project.org/docs/latest/tutorials/solver) (and the linked advanced tutorials)
* [Inverse Problem Example](http://phoebe-project.org/docs/latest/examples/inverse_paper_examples)
* [Passband Luminosity Tutorial](http://phoebe-project.org/docs/latest/tutorials/pblum)

Here you'll probably want to set `pblum_mode = 'dataset-coupled'` for your second and third light curves so that they reference the first.  The first can then use `pblum_mode = 'dataset-scaled'` as you start to explore the parameter space, but will eventually want to switch to `pblum_mode = 'component-coupled'` and _sample_ over the passband-luminosity when you do your final MCMC sampling (so that any parameter correlations are accounted for in your resulting posterior uncertainties).

Hope this helps you get started!
-Kyle
